### DISCUSSION TITLE
Does diff in ChartEntryModelProducer only shows differences on Y axis and not on X axis (scroll)?

### DISCUSSION THREAD

**Discussion by albertopasqualetto** [patrykandpatrick/vico#256] [Questions]
Does diff in ChartEntryModelProducer only shows differences on Y axis and not on X axis (scroll)?

I would like the graph to scroll at defined time intervals, but I want to see only a limited timespan in the graph.

**Comment by patrickmichalik** [comment]
Hello! Differences along the _x_-axis are not equivalent to scrolling, and `DefaultDiffProcessor` doesn’t animate such differences. Animated automatic scrolling can be added via `ChartScrollSpec`.
